core/src/commonMain/kotlin/io/github/aag/core/di/Koin.kt

```diff
@@ -11,9 +23,35 @@ import org.koin.dsl.module
 fun initKoin(appDeclaration: KoinAppDeclaration = {}) =
     startKoin {
         appDeclaration()
-        modules(commonModule())
+        modules(
+            commonModule(),
+            networkModule()
+        )
     }
 
 fun commonModule() = module {
     singleOf(::ProfileRepositoryImpl) bind ProfileRepository::class
 }
+
+internal fun networkModule() = module {
+    singleOf<HttpClient> {
+        HttpClient {
```

Comment:
Where is different http engine linked to platform?

Reply:
As far as I understand, ktor will choose and appropriate engine depending on artifacts:        
> You can also omit an engine:
> In this case, the client will choose an engine automatically depending on the artifacts
> https://ktor.io/docs/create-client.html